### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LarryKingStrivacity**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+
+See our [Developer Portal](https://www.strivacity.com/learn-support/developer-hub) to get started with developing for the Strivacity product.
+
+# Overview
+
+This SDK allows you to integrate Strivacity’s policy-driven journeys into your brand’s iOS mobile application. It implements Strivacity's no-code components via iOS's [ASWebAuthenticationSession](https://developer.apple.com/documentation/authenticationservices/aswebauthenticationsession) . 
+
+This SDK uses <https://appauth.io>, which follows the best practices from [RFC 8252 - OAuth 2.0 for Native Apps](https://tools.ietf.org/html/rfc8252),  including using in-app browser views like ASWebAuthenticationSession. Embedded user-agents, known as web-views, are not supported due to usability and security reasons documented in [Section 8.12 of RFC 8252](https://tools.ietf.org/html/rfc8252#section-8.12).
+
+The SDK uses the [PKCE extension to OAuth](https://tools.ietf.org/html/rfc7636), which secures authorization codes in public clients when custom URI scheme redirects are used.
```
Comment (bala-striva) [inline README.md]: this part is not related to the custom uri schema so i would remove this part:
```when custom URI scheme redirects are used.``` or replace it something like this ```when the clients is responsible for the token exchange```

Comment (LarryKingStrivacity) [reply README.md]: Updated and added to the commit.

**Review by bala-striva — APPROVED** [review]
